openapi/openapi-collections.ytt.yaml

```diff
@@ -61,6 +61,13 @@ paths:
           required: true
           schema:
             type: string
+        - name: timeout
+          in: query
+          description: |
+            Wait for operation commit timeout in millis. 
```

Comment:
It looks strange to specify timeout in millis while the default timeout if 10sec. Is there any practical reason to specify timeout less than 1sec, considering also that on some slow disks it might take more than a second to create collection on single node deployment.

Reply:
Yes, I agree, I will rewrite it to use seconds then